07-http_gordle/6_6_query_param/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.20
 require (
 	github.com/go-chi/chi/v5 v5.0.8
 	github.com/oklog/ulid/v2 v2.1.0
-	github.com/stretchr/testify v1.8.2
-	golang.org/x/exp v0.0.0-20230425010034-47ecfdc1ba53
```

Comment:
the other go.mod files weren't updated.
try `make go get -u ./...` at the root

Reply:
It's not working.... Can you test it?